### PR TITLE
1115 Distinguishes HC border from focus border, refactors to use common mixin

### PR DESCRIPTION
**Pull request by aleksandrjPersonal**
#### Pull request checklist

- [x] Addresses an existing issue: #1115
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

ChoiceGroup component: Distinguishes HC border from focus border, removes d
ouble stroke, refactors CommandBar to use common mixin

#### Focus areas to test
ChoiceGroup 
CommandBar

### CODE REVIEW

File: packages/office-ui-fabric-react/src/common/_focusBorder.scss [hunk 1]
```diff
@@ -37,3 +37,22 @@
     outline: 1px solid $focusedBorderColor;
   }
 }
+
+@mixin pseudo-border($color) {
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/common/_focusBorder.scss]: One last request. It seems that a focus border is nothing more than pseudo-border on :focus. Can we reuse the pseudo-border inside of focus-border and get the same css in return?

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt Refactored. Since the two mixins (pseudo-border and focus-border) used similar properties but with slightly different rules (including a :global) I moved the common values into two new mixins which are now used by the pseudo-border and focus-border. 

The new mixins are called base-border and after-border.

Please review :)

**Comment by aleksandrjPersonal** [comment]
1115_choicegrouphcfix
@battletoilet please review

**Comment by aleksandrjPersonal** [comment]
@battletoilet please review
1115_choicegrouphcfix

**Comment by battletoilet** [comment]
Remove border in high contrast mode.

**Comment by micahgodbolt** [comment]
I believe it'd be possible to just have 2 mixins, the focus-border (which includes pseudo-border upon focus). The relative and transparent styles only need to be applied upon focus. 

Is it possible to simplify this with just those 2 mixins?

**Comment by micahgodbolt** [comment]
I'd be happy to sit down with you tomorrow and walk through what I'm thinking (not in office today)

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt It may be possible to achieve an added level of nesting, but it was not obvious to me because of the various conflicting rules. Specifically I was unsure how one would fold ' :global(.ms-Fabric.is-focusVisible) &:focus:after' in to a simple :after -- I would be down to explore if you wanted to spend some time tinkering.

FYI I have updated some of the mixins based on feedback from the designer to include padding and thickness. This may or may not effect the tweaks you wanted to make.

**Comment by aleksandrjPersonal** [comment]
@battletoilet @micahgodbolt 
1115_choicegrouphcfix

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss [hunk 2]
```diff
@@ -26,7 +27,7 @@ $ms-choiceField-transition-timing: cubic-bezier(.4, 0, .23, 1);
   font-size: $ms-font-size-m;
   font-weight: $ms-font-weight-regular;
   min-height: 36px;
-  border: 1px solid transparent;
+  border: none;
```
Comment (aleksandrjPersonal) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss]: No border in HC mode

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss [hunk 3]
```diff
@@ -313,5 +315,9 @@ $ms-choiceField-transition-timing: cubic-bezier(.4, 0, .23, 1);
 }
 
 :global(.ms-Fabric.is-focusVisible) .choiceFieldIsInFocus {
-  border: 1px solid $ms-color-neutralPrimary;
+  @include pseudo-border($ms-color-neutralPrimary);
+}
+
+:global(.ms-Fabric.is-focusVisible) .choiceFieldIsInFocus.choiceFieldIsImage {
+  @include pseudo-border($ms-color-neutralPrimary, 2px, 1px);
```
Comment (aleksandrjPersonal) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss]: Adding padding and thickness vars

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt now using single mixin "focus-border" --  takes a Boolean $focus which runs an @if/@else to include rules for :after vs. focus:after

Please review

**Comment by micahgodbolt** [comment]
@aleksandrjPersonal looks like a good solution to me.

**Pull request merged**